src/MoBi.Core/Commands/RenameContainerCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using MoBi.Assets;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Assets;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Events;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Core.Commands
+{
+   public class RenameContainerCommand : BuildingBlockChangeCommandBase<ISpatialStructure>
```

Comment:
@Yuri05 Special command to rename containers in spatial structure. In this case, we need to also check if they are used in neighborhood and update the path